app/helpers/samples_helper.rb

```diff
@@ -62,6 +64,15 @@ def controlled_vocab_form_field(attribute, element_name, value)
     end
   end
 
+  def sample_multi_form_field(attribute, element_name, value)  
+    existing_objects = []
+    value.each {|v| existing_objects << Struct.new(:id, :name).new(v[:id], v[:title]) if v} if value
```

Comment:
`existing_objects` can also be a JSON string, so you can just pass `value.to_json` instead of converting to Structs

Reply:
I think since the value has **title** key rather than **name**, I can't pass the value.to_json

Reply:
I think in that case you can at least store the struct `Struct.new(:id, :name)` in a variable outside the loop, instead of making a new one for each iteration.